airflow/www/static/js/utils/index.ts

```diff
@@ -185,6 +185,34 @@ const toSentenceCase = (camelCase: string): string => {
   return "";
 };
 
+const addColorKeyword = (
+  parsedLine: string,
+  errorKeywords: string[],
+  warningKeywords: string[]
+): string => {
+  const lowerParsedLine = parsedLine.toLowerCase();
+  const containsError = errorKeywords.some((keyword) =>
+    lowerParsedLine.includes(keyword)
+  );
+  const bold = (line: string) => `\x1b[1m${line}\x1b[0m`;
+  const red = (line: string) => `\x1b[31m${line}\x1b[39m`;
```

Comment:
As the log might be long and this operation is potentially long-running - could you merge the bold+[red|yellow] operations into one to save some CPU?

Reply:
Thanks, it helped performance to go from 15ms to 2ms. I tried inlining by formatting and returning the string itself but for some reason the time went back to 15ms though it was my assumption inlining the code would increase performance which didn't happen on directly returning the string.

Reply:
okay, but you are measuring in ms still... most time probably is needed by browser then anyway.